app/models/practice_creator.rb

```diff
@@ -1,16 +1,29 @@
 class PracticeCreator < ApplicationRecord
   belongs_to :practice
   belongs_to :user, optional: true
+  after_create :process_crop
 
   acts_as_list scope: :practice
 
   # has_one :user, optional: true
-  has_attached_file :avatar
+  has_attached_file :avatar, styles: { thumb: '200x200>' }, :processors => [:cropper]
 
   attr_accessor :delete_avatar
+  attr_accessor :crop_x, :crop_y, :crop_w, :crop_h
 
   validates_attachment_content_type :avatar, content_type: %r{\Aimage/.*\z}
 
+  def process_crop
```

Comment:
i'm wondering if we can abstract this out for the different things we need to crop?
on the `ApplicationRecord` level